pkg/cmd/generate/util.go

```diff
@@ -159,6 +159,7 @@ const header = `# --------------------------------------------------------------
 func writeFile(filePath string, content []byte) error {
 	// https://github.com/kubernetes/kubernetes/issues/67610
 	contentString := strings.ReplaceAll(string(content), "\n  creationTimestamp: null", "")
+	contentString = strings.ReplaceAll(contentString, "\n      creationTimestamp: null", " {}")
```

Comment:
I would rather make it more generic so it replaces any empty creationTimestamp no matter at which level
```suggestion
	emptyCreationTimestamp := regexp.MustCompile(`\n *creationTimestamp: null`)
	contentString := emptyCreationTimestamp.ReplaceAllString(string(content), "")
```

Reply:
Took your suggestion here